<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotificationPreferences</class>
 <widget class="QWidget" name="NotificationPreferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>180</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="bounceTitleLabel">
     <property name="text">
      <string>If desktop notifications are enabled, bounce dock icon:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Maximum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>10</number>
       </property>
       <item>
        <widget class="QRadioButton" name="bounceOnAll">
         <property name="text">
          <string>On all notifications</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="bounceOnPrivate">
         <property name="text">
          <string>On private messages and mentions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="bounceNever">
         <property name="text">
          <string>Never bounce dock icon</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="webpageSettingsLabel">
     <property name="text">
      <string>Configure your desktop notification settings on your &lt;a href=&quot;#settings&quot;&gt;Settings&lt;/a&gt; page</string>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
